apps/prairielearn/src/components/AssessmentRegenerate.html.ts

```diff
@@ -16,10 +16,10 @@ export function RegenerateInstanceModal({ csrfToken }: { csrfToken: string }) {
     `,
     footer: html`
       <form method="POST">
+        <button type="button" data-dismiss="modal" class="btn btn-secondary">Cancel</button>
```

Comment:
Conventionally, this should come after the hidden inputs.

Reply:
This is not always consistent...
https://github.com/PrairieLearn/PrairieLearn/blob/7d9a602aff3a9ab945009517d0a0964c7bbc8f97/apps/prairielearn/src/components/GroupWorkInfoContainer.html.ts#L103-L106

Should I change those cases too?

Reply:
Done in 0c7b1f0.